.github/workflows/setup_docker_and_execute_command.yml

```diff
@@ -0,0 +1,24 @@
+name: Setup Docker and Execute Command
```

Comment:
i am thinking about a more "generic" name for this .yml script, i believe that in near future we will put here some more setup, but i don't have a better idea for now (as we already have here docker setup and google services file fetching)

Reply:
In the future, we can name it more generic when we will adding something new, and I think when we would have a lot of things we can just name it like workflow or something because still, we can name jobs.

Reply:
Probably we will just add new jobs to it 😅

Reply:
`setup_build_environment` should be enough for the file

Reply:
Done 😄